docs/Replicating-Alpaca.md

```diff
@@ -0,0 +1,99 @@
+# Replicating Alpaca
+
+In this tutorial, we will replicate [Stanford Alpaca](https://crfm.stanford.edu/2023/03/13/alpaca.html)
+using the new [Llama 2](https://ai.meta.com/llama/) model and [Levanter](https://github.com/stanford-crfm/levanter).
+We'll use a TPU V3-32 VM, though this same tutorial should work on an A100 box as well.
+
+## Setup
+
+### Cloning Levanter
+
+First, we'll clone Levanter:
+
+```bash
+git clone https://github.com/stanford-crfm/levanter.git
+cd levanter
+pip install -e .
+```
+
+### Setting up a TPU VM
+
+First, we'll spin up a TPU VM using the [Getting Started with TPUs](./Getting-Started-TPU-VM.md) guide.
+If you haven't gone through that guide before, you should do so now. If you have, you can just run, e.g.:
+
+```bash
+bash infra/spin-up-vm.sh llama-32 -z us-east1-d -t v3-32 --preemptible
+```
+
+## Choosing a Llama
+
+### Llama 1
+
+If you want, you can use [Decapoda's repo for Llama 1](https://huggingface.co/decapoda-research/llama-7b-hf),
+you'll just need to pass in `--model_name_or_path decapoda-research/llama-7b-hf` instead of `meta-llama/Llama-2-7b-hf`.
+
+### Getting Llama 2
+
+If you haven't already, go to [Llama 2's Hugging Face page](https://huggingface.co/meta-llama/Llama-2-7b-hf) and request access to the model.
+
+Once you have access, go to [Hugging Face's Tokens page](https://huggingface.co/settings/tokens) to get an API token.
+
+## The Alpaca script
+
+We have a [Levanter version](https://github.com/stanford-crfm/levanter/blob/main/examples/alpaca.py) of the [original Alpaca script](https://github.com/tatsu-lab/stanford_alpaca/blob/main/train.py)
```

Comment:
```suggestion
We have a [Levanter version](../examples/alpaca.py) of the [original Alpaca script](https://github.com/tatsu-lab/stanford_alpaca/blob/main/train.py)
```

Doesn't need to be in this way, but I feel relative link is cooler :)
I have tested locally to make sure the link works.

Reply:
it doesn't work with the mkdocs though :(